docs/coverage-reporter/uploading-coverage-in-advanced-scenarios.md

```diff
@@ -10,8 +10,10 @@ The following sections include instructions on how to use the Codacy Coverage Re
 
 If your test suite is split in different modules or runs in parallel, you must upload multiple coverage reports for the same language either at once or in sequence.
```

Comment:
I noticed that we had a small typo here while making the suggestion for reorganizing the intro:

```suggestion
If your test suite is split into different modules or runs in parallel, you must upload multiple coverage reports for the same language either at once or in sequence.
```